docs/source/tutorials/index.rst

```diff
@@ -0,0 +1,32 @@
+Tensor Comprehensions Tutorials
+===============================
+
+**Author**: `Priya Goyal <https://github.com/prigoyal>`_
+
+Tensor Comprehensions (TC) is a framework agnostic library to **automatically**
+synthesize high-performance Machine Learning kernels. TC relies on
+`Halide <https://github.com/halide/Halide>`_ IR to express computation and analysis
+tools to reason about it. TC uses :code:`polyhedral` compilation techniques to
+(semi-)automatically decide how to perform this computation efficiently and produce
+fast code. We also provide TC integration with PyTorch and Caffe2.
+
+To read more about Tensor Comprehensions, see our documentation available
+at https://facebookresearch.github.io/TensorComprehensions/ and C++ API documentation is
+available at https://facebookresearch.github.io/TensorComprehensions/api.
+
+We provide many **python examples** for expressing and running various different ML layers
+with TC. The examples can be found `here <https://github.com/facebookresearch/TensorComprehensions/tree/master/test_python/layers>`_.
+
+To read more about Framework integrations, checkout our documentation on `PyTorch <https://facebookresearch.github.io/TensorComprehensions/framework/pytorch_integration/getting_started.html>`_ integration
+and `Caffe2 <https://facebookresearch.github.io/TensorComprehensions/framework/caffe2_integration/integration_with_example.html>`_
+integration.
+
+If you want to **integrate your framework** with TC, it's easy and the instructions are
+available at https://facebookresearch.github.io/TensorComprehensions/integrating_any_ml_framework.html
```

Comment:
This link does not seem to work for me, is it supposed to already?
I do not see another mention of ```integrating_any_ml_framework``` in this PR so I am wondering wheterh there is a dependent PR or whether this needs to be changed.

Reply:
yep, it needs to be hosted, after the PR lands, I will regenerate the docs and tutorial and this link should work then.